### DISCUSSION TITLE
Cuales son los comandos de consola

### DISCUSSION THREAD

**Discussion by reinierssp** [TEHE-Studios/ExpandedHelicopterEvents#178] [Q&A] [answered]
Buenas tardes me gustaría conoces los comandos de consola para activar los eventos aunque sea ramdom pero poder forzar el inicio de uno en mi servidor MP

**Comment by Chuckleberry-Finn** [comment, marked as answer]
You can log into your server in debug mode if you wish to use the launch test panel. Otherwise you can use the twitch sub-mod.

**Reply by reinierssp** [reply]
mychisimas gracias y disculpe las molestias y la demora
